deploy/helm/sumologic/values.yaml

```diff
@@ -56,6 +56,7 @@ sumologic:
     # config_context_auth_info:
     # config_context_cluster:
     token: "${file(\"/var/run/secrets/kubernetes.io/serviceaccount/token\")}"
+    ## load_config_file is deprecated and no longer used
```

Comment:
Remove it in 3.0

Reply:
I would just remove this `sumologic.cluster.load_config_file` property from `values.yaml` instead of commenting that it doesn't work.